### DISCUSSION TITLE
How to preselect rows of a qtable

### DISCUSSION THREAD

**Discussion by shi-yan** [quasarframework/quasar#12510] [General - Components / Directives / etc]
how to preselect qtable rows?

I tried setting the selected property of the qtable component, but it doesn't work.

https://quasar.dev/vue-components/table#introduction
```javascript

sarr = [{....}]

<q-table
...
:selected = "sarr"
 />
```

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/VwrQeEM?editors=1010](https://codepen.io/milospaunovic/pen/VwrQeEM?editors=1010)

If this doesn't guide you in the direction of solving the problem, please create a Codepen reproduction so it would be easier for someone to help you.
